pkg/controllers/resources/persistentvolumes/fake_syncer_test.go

```diff
@@ -3,13 +3,13 @@ package persistentvolumes
 import (
 	"testing"
 
-	synccontext "github.com/loft-sh/vcluster/pkg/controllers/syncer/context"
+	"github.com/loft-sh/vcluster/pkg/syncer/synccontext"
+	testing2 "github.com/loft-sh/vcluster/pkg/syncer/testing"
```

Comment:
can't we give this a better name? something like syncertesting maybe?

Reply:
Yep will do